contribs/gmf/src/services/themesservice.js

```diff
@@ -188,7 +188,7 @@ gmf.Themes.findObjectByName_ = function(objects, objectName) {
  * Find a theme object by its name. Return null if not found.
  * @param {Array.<gmfThemes.GmfTheme>} themes Array of "theme" objects.
  * @param {string} themeName The theme name.
- * @return {gmfThemes.GmfTheme} The theme object.
+ * @return {?gmfThemes.GmfTheme} The theme object or null.
```

Comment:
That explain why that doesn't change anything. I change the `?` but keep the ` or null` (because without that, some people (me included) forget that the result can be null)